cmd/corectl/main.go

```diff
@@ -68,6 +68,8 @@ var commands = map[string]*command{
 	"get":                  {get},
 	"add":                  {add},
 	"rm":                   {rm},
+	"set":                  {set},
+	"unset":                {unset},
```

Comment:
Good q. Would it take the current value as an argument or just be `corectl rm <key>`? I don't really have an opinion either way in unset vs rm.